packages/cxl-lumo-styles/scss/typography.scss

```diff
@@ -0,0 +1,129 @@
+html,
+:host {
+  /* Font families */
+  --lumo-font-family: "Roboto", Arial, sans-serif;
+  --lumo-font-secondary: "Libre Baskerville", Georgia, serif;
+  /* Font sizes */
+  --lumo-font-size-xxs: 0.75rem;
+  --lumo-font-size-xs: 0.8125rem;
+  --lumo-font-size-s: 0.875rem;
+  --lumo-font-size-m: 1rem;
+  --lumo-font-size-l: 1.125rem;
+  --lumo-font-size-xl: 1.375rem;
+  --lumo-font-size-xxl: 1.75rem;
+  --lumo-font-size-xxxl: 2.5rem;
+  /* Line heights */
+  --lumo-line-height-xs: 1.25;
+  --lumo-line-height-s: 1.375;
+  --lumo-line-height-m: 1.625;
+  /* FYI @leho @jaan */
+  --lumo-border-radius-s: 0.25em;
+  --lumo-border-radius-m: 0.5em;
+  --lumo-border-radius-l: 0.75em;
+}
+
+html {
+  font-family: var(--lumo-font-family);
+  font-size: var(--lumo-font-size, var(--lumo-font-size-m));
+  line-height: var(--lumo-line-height-m);
+  -webkit-text-size-adjust: 100%;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+/* Can’t combine with the above selector because that doesn’t work in browsers without native shadow dom */
+:host {
+  font-family: var(--lumo-font-family);
+  font-size: var(--lumo-font-size, var(--lumo-font-size-m));
+  line-height: var(--lumo-line-height-m);
+  -webkit-text-size-adjust: 100%;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+small,
+[theme~="font-size-s"] {
+  font-size: var(--lumo-font-size-s);
+  line-height: var(--lumo-line-height-s);
+}
+
+[theme~="font-size-xs"] {
+  font-size: var(--lumo-font-size-xs);
+  line-height: var(--lumo-line-height-xs);
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-weight: 600;
+  line-height: var(--lumo-line-height-xs);
+  margin-top: 1.25em;
+}
+
+h1 {
+  font-size: var(--lumo-font-size-xxxl);
+  margin-bottom: 0.75em;
+}
+
+h2 {
+  font-size: var(--lumo-font-size-xxl);
+  margin-bottom: 0.5em;
+}
+
+h3 {
+  font-size: var(--lumo-font-size-xl);
+  margin-bottom: 0.5em;
+}
+
+h4 {
+  font-size: var(--lumo-font-size-l);
+  margin-bottom: 0.5em;
+}
+
+h5 {
+  font-size: var(--lumo-font-size-m);
+  margin-bottom: 0.25em;
+}
+
+h6 {
+  font-size: var(--lumo-font-size-xs);
+  margin-bottom: 0;
+  text-transform: uppercase;
+  letter-spacing: 0.03em;
+}
+
+p,
+blockquote {
+  margin-top: 0.5em;
+  margin-bottom: 0.75em;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+hr {
+  display: block;
+  align-self: stretch;
+  height: 1px;
+  border: 0;
+  padding: 0;
+  margin: var(--lumo-space-s) calc(var(--lumo-border-radius-m) / 2);
+  background-color: var(--lumo-contrast-10pct);
+}
+
+blockquote {
+  border-left: 2px solid var(--lumo-contrast-30pct);
+}
+
+b,
+strong {
+  font-weight: 600;
+}
```

Comment:
Are these really all our overrides?

Reply:
what you can see in typography.scss is override